src/NzbDrone.Core/Datastore/Migration/221_add_season_pack_upgrade.cs

```diff
@@ -0,0 +1,19 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(221)]
+    public class add_season_pack_upgrade : NzbDroneMigrationBase
```

Comment:
This migration isn't necessary, the default value can be persisted in the config automatically.